Modules/MUON/Common/src/MergeableTH1Ratio.cxx

```diff
@@ -115,4 +115,11 @@ void MergeableTH1Ratio::update()
   }
 }
 
+void MergeableTH1Ratio::Reset(Option_t* option)
+{
+  getNum()->Reset(option);
```

Comment:
do we have a guarantee that getNum() won't return a nullptr ?

Reply:
The num and den histograms are initialized in the constructors. They could be null if there is not enough memory, but then things would break much earlier, for example during the merge or update, which are also not protected. I can however add protections everywhere just to be on the safe side...

Reply:
well, the default constructor does not create histograms, does it ? So basically if you do : 

```
MergeableTH1Ratio h;
h.Reset();
```

you'd get a crash, right ?

Reply:
You have a good point here...